lib/writer.js

```diff
@@ -14,7 +18,6 @@ function writeToFS(components, options, moduleFileNameDelimiter) {
   options = options || {};
 
   var outPath = options.path || 'components';
-  var delimiter = moduleFileNameDelimiter || '-';
```

Comment:
To fix wrong filename when calling via API you may want to leave this, but change alternative value to empty string `''`.